language/de/acp/forums.php

```diff
@@ -132,6 +132,8 @@
 	'GENERAL_FORUM_SETTINGS'	=> 'Allgemeine Forumseinstellungen',
 
 	'LINK'						=> 'Link',
+	'LIMIT_SUBFORUMS'			=> 'Beschränke die Legende auf die direkten Unterforen',
+	'LIMIT_SUBFORUMS_EXPLAIN'	=> 'Beschränkt die Unterforenanzeige auf die Unterforen, die direkte Nachfragen (Kinder) des aktuellen Forums sind. Deaktiviert werden alle Unterforen mit der aktivierten Option „Unterforen in Legende aufführen“ unabhängig von deren Tiefe angezeigt.',
```

Comment:
...die Anzeige der Unterforen auf die, die...
Nachfragen  -> Nachfahren

Deaktiviert werden alle Unterforen mit der aktivierten Option „Unterforen in Legende aufführen“ unabhängig von deren Tiefe angezeigt. -> Ist diese Einstellung deaktiviert, so werden alle Unterforen unabhängig deren Tiefe angezeigt, bei denen die Option „Unterforen in Legende aufführen“ aktiviert ist.

Reply:
Ich finde das ganze irgendwie insgesamt etwas hölzern. Neu:
> Beschränkt die angezeigten Unterforen auf die direkten Kinder des jeweiligen Forums. Kindeskinder werden nicht angezeigt. Ist diese Einstellung deaktiviert, so werden alle Unterforen unabhängig von deren Verschachtelungstiefe angezeigt, bei denen die Option „Unterforen in Legende aufführen“ aktiviert ist.

Evtl. auch den "Kindeskinder.." Satz weglassen.

Reply:
Passt für mich